src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -399,17 +400,22 @@ private function modifyAllLayers(ImageInterface $image, callable $modifier)
             $layers = [];
         }
 
-        if (count($layers)) {
+        if (count($layers) > 1) {
```

Comment:
Did debug and this will save us a lot of memory 40mb -> 8mb tested on the imagick adapter.

Reply:
@danrot backported this to 1.6: https://github.com/sulu/sulu/pull/4968

Reply:
I would prefer to not have this change in here - then it is less likely that there'll be a conflict when I merge back, and I have less to test in this PR :-)

Reply:
@danrot tests will fail then as the clone doesn't work on `GD` driver.

Reply:
But doesn't that mean that it will fail if you upload an animated GIF if using GD in general? :thinking:

Reply:
GD doesn't support animated gifs or other multi layer image formats [see](https://imagine.readthedocs.io/en/stable/usage/layers.html#disclaimer).